external/CMakeLists.txt

```diff
@@ -27,7 +27,6 @@ if(GISMO_BUILD_CPPLOT)
 add_subdirectory(cpplot)
 endif(GISMO_BUILD_CPPLOT)
 
-#add_subdirectory(Eigen)
 #add_subdirectory(rapidxml)
```

Comment:
It would be nice to also remove external/gzstream and fetch it from
https://gitlab.inria.fr/gismo/gzstream

And also zlib from https://github.com/madler/zlib